### PR TITLE
(OUTDATED) Add workshop documentation

### PR DESCRIPTION
**Pull request by Equalizer5118**
This pr will contain all the documentation on the current state of the workshop in p2ce. More pages to come, this is fairly close to being done (!!!).

### CODE REVIEW

**Comment by JJL772** [comment]
Thanks for this! Keep in mind that we'll probably change quite a bit as the implementation of workshop matures. I wouldn't bother going into detail about the quirks just yet.

I'll let you know when we're nearing public release, then we'll work with you to document the nitty-gritty details and issues with our implementation.

File: pages/shared/reference/materials/materialcreation.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+### Method 1: GIMP
+GIMP can combine different layers into the RGBA channels of one texture, which makes creating MRAO maps using GIMP trivial. 
+
+First, go to "Image -> Mode" and make sure "Grayscale" is selected. Even if your images are grayscale, GIMP may still treat them as color images, which prevents this method from working. Then, create a new image with the dimensions of your texture maps. Go to file and click "Import as layers...", select your Metalness, Roughness, and AO maps, and click Import. Your layers should now look similar to this (order is irrelevant):
+
+![Metalness, Roughness, and AO maps in separate layers](https://github.com/Equalizer5118/Wiki/assets/113145248/035e9064-c4f1-4a76-bf3e-8ae822680c18)
+
+After this, go to "Colors -> Components -> Compose...". Put your Metalness map in the Red channel, your Roughness map in the Green, and your AO map in the Blue. The dropdowns should look like this:
+
+![Metalness map in R, Roughness in G, AO in B](https://github.com/Equalizer5118/Wiki/assets/113145248/54573338-4810-4dbe-b3f0-ca74541c9aa8)
```
Comment (JJL772) [inline pages/shared/reference/materials/materialcreation.md:31]: These paths need to be updated, I don't see the assets included in this PR.

File: pages/shared/reference/materials/pbrshader.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+#title: The PBR Shader
+---
+This page documents all the shader parameters of the PBR shader in all Strata games.
+(TODO: Document this!)
```
Comment (JJL772) [inline pages/shared/reference/materials/pbrshader.md:5]: I've got a little shader documentation markdown generator that could be used as the basis for this page.

https://github.com/JJL772/strata-shader-docs/releases/download/v2024.1.1-0/shaders.zip

**Review by JJL772 — CHANGES_REQUESTED** [review]
Once again, really appreciate your work here! Just a few comments and misc notes left :)

Main thing is the asset paths in the PBR docs. we can go ahead and merge after that's fixed.

File: pages/p2ce/reference/workshop/addoncreation.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+### Using workshopgui
+In the `bin\win64` folder of the root install, towards the bottom of the lists you will find both workshop uploaders. Open the one labeled `workshopgui.exe`. After a short wait, a small window with a large button should appear in the center of your screen. Click on this button, and a file explorer window should appear. Navigate to your addon folder, select it, and click the "Select Folder" button. After a short wait, a notice saying that `addon.kv3` is missing, and that it will automatically create one for you; click ok to do this. The program will then freeze for about a minute, then open up your newly uploaded addon on your browser of choice. Don't worry, it's hidden, so nobody can see it!
```
[the remark this answers is not in the document]
Comment (Equalizer5118) [reply pages/p2ce/reference/workshop/addoncreation.md:13]: Alright, when the new uploader ships ill document it

File: pages/shared/reference/materials/pbrshader.md (hunk 2 above)
Comment (Equalizer5118) [reply pages/shared/reference/materials/pbrshader.md]: Basically ported the contents of the generated page underneath my text. The VDC splits parameters into their own subheadings for readability (see [LightmappedGeneric](https://developer.valvesoftware.com/wiki/LightmappedGeneric)), so I may do that later. Its usable atm, however.

File: pages/shared/reference/materials/materialcreation.md (hunk 1 above)
Comment (Equalizer5118) [reply pages/shared/reference/materials/materialcreation.md]: I ported the images into their own folder in [/assets/PBR_images](https://github.com/Equalizer5118/Wiki/tree/patch-1/assets/PBR_images), and added them into the article in what I think should be fine. I more than likely did something wrong, so I wouldn't be surprised if its broken

File: pages/p2ce/reference/workshop/addoncreation.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+### Using workshopcli 
+*TODO: Document this*
```
[the remark this answers is not in the document]
Comment (Equalizer5118) [reply pages/p2ce/reference/workshop/addoncreation.md]: Completed documentation on this in the newest revision

File: pages/p2ce/reference/workshop/addoncreation.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Equalizer5118) [reply pages/p2ce/reference/workshop/addoncreation.md:14]: Still need to get this done, I'll do it today

**Comment by Equalizer5118** [comment]
Closed PR due to being heavily outdated, also in favor or new docs seen [here](https://github.com/StrataSource/Wiki/pull/79)

**Pull request closed without merging**